### ISSUE TITLE
Multi-word and multi-line keywords

### ISSUE THREAD

**Issue by josusky** [wmo-im/wcmp#100]
**Summary and Purpose**
Current version of the [KPI-6](https://github.com/wmo-im/wcmp/blob/master/kpi/006-keywords.adoc) verifies several aspects of keyword elements but not the actual keyword values. Preliminary tests with pywcmp show, that there are records with keywords that are whole sentences or even have multiple lines.

**Proposal**
1. Keywords containing line break shall be considered invalid. Found examples look like lists of keywords that were incorrectly divided when creating the XML representation.
2. Multi word keywords (sic!) cannot be systematically banned as it is common practice to put observing station name as a geographic keyword. Further discussion is needed to find out if _single word keywords_ could be a rule for other types.

**Reason**
Keywords are key elements for data discoverability therefore we should make them reliable.

**Comment by amilan17** [comment]
@josusky - is this still an open issue?

**Comment by josusky** [comment]
Yes, this is still an issue. For example "urn:x-wmo:md:ca.gc.ec::1.1.1.3" contains following definition of keywords:
```
              <gmd:descriptiveKeywords>
                <gmd:MD_Keywords>
                  <gmd:keyword>
                    <gco:CharacterString>Wind Velocity
Geopotential Height
Pressure Levels</gco:CharacterString>
                  </gmd:keyword>
                </gmd:MD_Keywords>
              </gmd:descriptiveKeywords>

```
Apparently, this should have been 3 keywords.
On the other hand, for example `urn:x-wmo:md:int.wmo.wis::CUJD01OJAM` (urn_x-wmo_md_int.wmo.wis__CUJD01OJAM.xml) contains definition of keywords that starts with an empty keyword (that has a thesaurus):
```
            <gmd:descriptiveKeywords>
                <gmd:MD_Keywords id="WMOCodeListKeywords">
                    <gmd:keyword gco:nilReason="unknown" />
                    <gmd:type>
                        <gmd:MD_KeywordTypeCode codeList="http://standards.iso.org/ittf/PubliclyAvailableStandards/ISO_19139_Schemas/resources/Codelist/gmxCodelists.xml#MD_Keyword
TypeCode" codeListValue="theme">theme</gmd:MD_KeywordTypeCode>
                    </gmd:type>
                    <gmd:thesaurusName>
                        <gmd:CI_Citation id="WMOCodelist">
                            <gmd:title>
<gco:CharacterString>WMO_CodeList dictionary [http://wis.wmo.int/2010/metadata/version_1-2/WMOCodelists.xml#WMO_CategoryCode]</gco:CharacterString>
                            </gmd:title>
                            <gmd:date>
```
I have run the check on 49612 records (I am struggling to get it run over the full catalogue) and found 274 problematic keyword entries. That is not a lot, but it is still and issue.

**Comment by amilan17** [comment]
@josusky @tomkralidis -- I don't think we added these tests to the KPIs. Shall we close this issue and mark it as not done?
